### ISSUE TITLE
Pi-hole as DNS server loses internet, but I can connect to admin interface

### ISSUE THREAD

**Issue by raviriley** [pi-hole/pi-hole#1558]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**
1
_{replace this text with a number from 1 to 10, with 1 being not familiar, and 10 being very familiar}_

---
**[BUG REPORT | OTHER]:**
I can connect to Pi-hole admin interface but when using the pi-hole IP as DNS server, I lose internet access. 
Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behavior:**
Internet Access

**[BUG | ISSUE] Actual Behavior:**
No Internet Access

**[BUG | ISSUE] Steps to reproduce:**

-install pi-hole with wlan0 and static ip
-connect to pi-hole admin interface (working)
-use pi-hole as DNS server (for router or individual devices, both don't work)
-lose internet access

**(Optional) Debug token generated by `pihole -d`:**
ubp8nipzth
-this wasn't showing up because I accidentally wrote <> around it
`<token>`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by dschaper** [comment]
Please run `pihole -d` and provide the token generated. Thank you.

**Comment by raviriley** [comment]
@dschaper See above

**Comment by dschaper** [comment]
Looks like the interface is set to `sit0` which doesn't have IP connectivity, but the FTL engine is showing 3 clients connected. Try running `pihole -r` to reconfigure and set it to the `wlan0` interface.

**Comment by raviriley** [comment]
@dschaper what is `sit0`? I already tried the reconfiguring process 3 times with wlan0.

**Comment by raviriley** [comment]
@dschaper what is `sit0`? I already tried the reconfiguring process 3 times with wlan0.

**Comment by dschaper** [comment]
If you look at `/etc/pihole/setupVars.conf` you will see the interface was set to use `sit0`. Try running `ip link` and see if you have a `sit0` interface.

**Comment by jacobsalmela** [comment]
@raviriley did `pihole -r` fix the issue?  Did @dschaper's last comment provide any insight?

**Comment by raviriley** [comment]
@dschaper @jacobsalmela I used `nano /etc/pihole/setupVars.conf`and changed sit0 to wlan0. When I use my CHIP's static IP as the DNS server for my router, it still is the same. I can connect to the admin console but I can't access the internet from other devices. I am using openDNS servers 208.67.222.222 and 208.67.220.220. After manually replacing 'sit0' with 'wlan0' , I ran 'pihole -r' again. Then I reset my router and put in the DNS server as my static IP again. IT WORKED! WOOOO ad free internet here I come! Thanks for your help.
